app/views/articles/_podcast_episodes_feed.html.erb

```diff
@@ -18,6 +18,12 @@
       </div>
       <h4>
         <%= episode.podcast.title %>
+        <% if episode.published_at.present? %>
+          ・<time class="published-at" datetime="<%= episode.decorator.published_timestamp %>">
```

Comment:
`decorator` doesn't seem like a valid method, so rendering this template should fail.
I checked the codebase and found out that it's not used in the codebase. I suppose we should delete it. Please, tell me if you find that it's used and I haven't noticed it.